.github/workflows/autofix.ci.yaml

```
@@ -12,16 +12,26 @@ jobs:
+        with:
+          run_install: false
+
+      - name: Setup Node with pnpm cache
+        uses: actions/setup-node@v4
         with:
           node-version: 22
+          cache: 'pnpm'
           token: ${{ secrets.GITHUB_TOKEN }}
-      - name: Install
-        uses: ./.github/actions/install
+      
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)